expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java

```diff
@@ -83,17 +83,16 @@ private ExpandedPerson toExpandedPerson(String response, User owner) throws Json
     @Override
     public ExpandedDataEntry expandEntry(Entity dataEntry) throws JsonProcessingException, NotFoundException {
         var identifier = dataEntry.getIdentifier();
-        if (dataEntry instanceof Resource) {
-            logger.info("Expanding Resource: {}", identifier);
+        if (dataEntry instanceof Resource resource) {
+            logger.info("Expanding Resource: {}", resource.getIdentifier());
             var expandedResource = ExpandedResource.fromPublication(uriRetriever,
-                                                                 dataEntry.toPublication(resourceService));
+                                                                 resource.toPublication(resourceService));
             return NviCalculator.calculateNviType(expandedResource);
-        } else if (dataEntry instanceof TicketEntry) {
-            logger.info("Expanding TicketEntry: {}", identifier);
+        } else if (dataEntry instanceof TicketEntry ticketEntry) {
+            logger.info("Expanding TicketEntry: {}", ticketEntry.getIdentifier());
             return ExpandedTicket.create((TicketEntry) dataEntry, resourceService, this, ticketService);
-        } else if (dataEntry instanceof Message) {
+        } else if (dataEntry instanceof Message message) {
             logger.info("Expanding Message: {}", identifier);
```

Comment:
message.getIdentier() to match other if statements, identifier variable on the top of method can be removed now.